cmd/lotus-shed/sectors.go

```diff
@@ -88,7 +88,7 @@ var terminateSectorPenaltyEstimationCmd = &cli.Command{
 		},
 	},
 	Action: func(cctx *cli.Context) error {
-		if cctx.NArg() < 1 {
+		if cctx.NArg() != 1 {
```

Comment:
no, see `cctx.Args().Slice()` below, undo this one and we don't need to be messing with the lotus-shed commands